README.md

```diff
@@ -570,6 +570,51 @@ Displays a leaderboard sorted by funds raised (highest first) for a single speci
 </html>
 ```
 
+#### MMF Leaderboard (Campaign)
+
+Displays a leaderboard sorted by distance, recorded using MapMyFitness, for a single specified campaign. Ordered by highest first. This leaderboard also displays, and can be sorted by, funds raised. Can be set to show leaderboards for teams or individuals.
+
+- `campaignUid`: *required* string campaign uid to filter results by campaign.
+- `type`: *optional* string. Set a type of either `'team'` or `'individual'`. Set to `'individual'` by default.
+- `limit`: *optional* number. Set to `100` by default. Determines how many results should be fetched via the everdayhero leaderboards API endpoint.
+- `pageSize`: *optional* number. Set to `5` by default. Determines how many results to display on the leaderboard at once.
+- `backgroundColor`: *optional* string. Set to `''` by default.
```

Comment:
Perhaps "Default is none." or "Not set by default." ?   "Set to `''`" looks a bit funny.